src/ejercicio3/Main.java

```diff
@@ -0,0 +1,68 @@
+package ejercicio3;
+
+import Utility.FileManager;
+import ejercicio3.clases.Pelicula;
+
+import java.util.ArrayList;
+
+public class Main {
+    public static void main(String[] args) {
+        final var listaPeliculas =
+                FileManager.leerFile("src/ejercicio3/input/informacionPeliculas.txt");
+
+        final var accionList = new ArrayList<String>();
+        final var comediaList = new ArrayList<String>();
+        final var terrorList = new ArrayList<String>();
+
+        generarData(listaPeliculas, accionList, comediaList, terrorList);
+
+        FileManager.escribirFile("src/ejercicio3/output/accion.txt", accionList);
+        FileManager.escribirFile("src/ejercicio3/output/comedia.txt", comediaList);
+        FileManager.escribirFile("src/ejercicio3/output/terror.txt", terrorList);
+    }
+
+    static void generarData(ArrayList<String> fileLeido,
+                            ArrayList<String> accion,
+                            ArrayList<String> comedia,
+                            ArrayList<String> terror) {
+
+        for (String string : fileLeido) {
```

Comment:
for(var sttring: fileLeido)

Reply:
fileLeido suena a un file, cambia por listaInput o listaLeida para que quede más claro